core/block.js

```diff
@@ -1280,6 +1280,7 @@ Blockly.Block.prototype.interpolate_ = function(message, args, lastDummyAlign) {
         }
       } while (altRepeat);
       if (field) {
+        field.fieldType = element['type'].replace(/field_/, '');
```

Comment:
the replacement is just to shave some extra text from the attribute. There's no explicit reason to remove it